src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started work");
```

Comment:
The message 'Truck started work' could be more informative. According to the checklist, it should indicate the type of `Machine` that is working. Consider specifying that it's a Truck that has started its work, for example: 'Truck is now working.'